src/sink/udp.rs

```diff
@@ -55,7 +71,20 @@ impl ConfigImpl for Config {}
 impl offramp::Impl for Udp {
     fn from_config(config: &Option<OpConfig>) -> Result<Box<dyn Offramp>> {
         if let Some(config) = config {
-            let config: Config = Config::new(config)?;
+            let mut config: Config = Config::new(config)?;
+            if !config.bound {
+                warn!("The `bound` setting of the UDP offramp is depricated, in futur re-binding will only work over the $udp metadata")
+            }
+            if let Some(dst_port) = config.dst_port.take() {
+                warn!("The `dst_port` setting of the UDP offramp is depricated, this will in future be `port` and `port` will be `bind.port`.");
```

Comment:
```suggestion
                warn!("The `dst_port` setting of the UDP offramp is deprecated, this will in future be `port` and current `port` will be `bind.port`.");
```